src/index.js

```diff
@@ -76,10 +76,26 @@ const getEntity = (entityOrId, schemaKey, entities, isImmutable) => {
     entities[schemaKey][entityOrId];
 };
 
-const getEntities = (entities, isImmutable) => (schema, entityOrId) => {
-  const schemaKey = schema.key;
-
-  return getEntity(entityOrId, schemaKey, entities, isImmutable);
+const getEntities = (entities, cache, isImmutable) => (schema, entityOrId) => {
+  return (handleCacheHit, handleCacheMiss) => {
+    const schemaKey = schema.key;
+    const getEntityId = (entity) => typeof entity === 'object' ? schema.getId(entity) : entity;
```

Comment:
Currently you're creating this function on every iteration. Unless I'm missing something, I think you could just set `entityId` at the top of the function and forego this function at all.

```js
return (handleCacheHit, handleCacheMiss) => {
  const schemaKey = schema.key;
  const entityId = typeof entityOrId === 'object' ? schema.getId(entityOrId) : entityOrId;
  // ...
```

If there is a reason for this method to be called each time, I'd create it b/w line 79/80.